arch/arm/boot/dts/overlays/i2c-rtc-common.dtsi

```diff
@@ -243,6 +243,21 @@
 		};
 	};
 
+	fragment@18 {
+		target = <&i2c_arm>;
```

Comment:
This line needs to say `<&i2cbus>`, like the others, otherwise it won't work in the i2c-rtc-gpio overlay.